runtime/runtime.go

```diff
@@ -289,6 +289,14 @@ func (rt *Runtime) StartServer(ctx context.Context) {
 // will block until either: an error occurs, the context is canceled, or
 // a SIGTERM or SIGKILL signal is sent.
 func (rt *Runtime) Serve(ctx context.Context) error {
+
+	undo, err := maxprocs.Set()
+	if err != nil {
+		logrus.WithField("err", err).Debug("Failed to set GOMAXPROCS based on environment.")
+	}
+
+	defer undo()
```

Comment:
💭 It doesn't get us anything, I believe, but it also won't hurt...

For some reason, the code examples of the library put the `defer undo()` right before the error check. Weird.